apps/src/sites/studio/pages/levels/_free_response.js

```diff
@@ -1,18 +1,39 @@
+/* global dashboard appOptions */
 import $ from 'jquery';
 import React from 'react';
-import ReactDom from 'react-dom';
+import ReactDOM from 'react-dom';
```

Comment:
Renamed `ReactDom` to `ReactDOM` to follow our own usage elsewhere and [usage in the React documentation](https://reactjs.org/docs/react-dom.html).